package.json

```diff
@@ -32,7 +35,8 @@
     "jest": "^21.2.1",
     "prettier": "~1.7.4",
     "rollup": "^0.50.0",
+    "typescript": "^2.5.3",
     "uglify-js": "^2.7.5"
   },
-  "typings": "picodom.d.ts"
+  "typings": "./src/picodom.d.ts"
```

Comment:
@andrewiggins Can we move picodom.d.ts to the root of the project?

Reply:
@JorgeBucaran reverted my change and added "picodom.d.ts" to the package.json `files` list so it is included in the npm package.